src/components/MenuPanel/MenuPanel.js

```diff
@@ -105,8 +120,52 @@ class MenuPanel extends React.PureComponent {
                     : this.renderAppGroup(app, false)
                 )}
               </div>
-              <h1 style={{ marginTop: '24px' }}>System</h1>
-              <div>{systemApps.map(app => this.renderAppGroup(app, true))}</div>
+              <FocusVisible>
+                {({ onFocus, focusVisible }) => (
+                  <StyledButton
+                    onFocus={onFocus}
+                    focusVisible={focusVisible}
+                    onClick={this.handleToggleSystemApps}
+                  >
+                    <h1
+                      style={{
+                        marginTop: '24px',
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        alignItems: 'flex-end',
+                      }}
+                    >
+                      <span>System</span>
+                      <span
+                        css={`
+                          font-size: 7px;
+                          opacity: 0.7;
+                        `}
+                      >
+                        {systemAppsOpened ? '▼' : '▲'}
+                      </span>
+                    </h1>
+                  </StyledButton>
+                )}
+              </FocusVisible>
+              <Transition
+                items={systemAppsOpened}
+                config={springs.lazy}
```

Comment:
Can we try [`springs.swift`](https://github.com/aragon/aragon-ui/blob/master/src/utils/styles/spring.js#L17) instead? `lazy` is meant to be used for large objects.